tasks/main.yml

```diff
@@ -56,5 +86,5 @@
 - name: start and enable postfix
   service:
     name: "{{ postfix_service }}"
-    state: started
+    state: restarted
```

Comment:
This would not become idempotent

Reply:
as long as state says restarted i can run the playbook multiple times over a deployed vm
with all new settings applied as the templates and any aliases or other files that need a restart would get applied.

if you leave it on start only:

changes to the config file do not get applied or anything above like aliases.
I'm looking at the code where something would break idempotency, but can't find any
except for the service that needs to be restarted

Reply:
ok latest commit should fix the handlers to fire at the right time and right order to get the hash files loaded correctly